<?xml version='1.0' encoding='UTF-8'?>
<defects4j>
	<project projectid="Chart">
		<bug bugid="Chart-19">
			<tests>
				<testfile path="org/jfree/chart/plot/junit/CategoryPlotTests">
					<exception>junit.framework.AssertionFailedError:-
						org.jfree.chart.plot.junit.CategoryPlotTests::testGetDomainAxisIndex:--&gt;
						junit.framework.AssertionFailedError</exception>
				</testfile>
				<spectra>
					<number_tests>366</number_tests>
					<failed>2</failed>
					<passed>364</passed>
					<nodes>44608</nodes>
				</spectra>
			</tests>
			<fixlocations>
				<file path="org/jfree/chart/plot/CategoryPlot.java">
					<insert numberlines="3">698</insert>
					<insert numberlines="3">973</insert>
					<bugtypes>
						<id id="1.3" lines="697">
							<description>Missing exception handling for null value.</description>
						</id>
						<id id="1.3" lines="972">
							<description>Missing exception handling for null value.</description>
						</id>
					</bugtypes>
				</file>
			</fixlocations>
			<numberfixedlines>2</numberfixedlines>
		</bug>
	</project>
</defects4j>
